lib/graphql/persisted_queries/builder_helpers.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module GraphQL
+  module PersistedQueries
+    # Contains factory methods for error handlers
+    module BuilderHelpers
+      def self.camelize(name)
```

Comment:
This looks like a perfect case for a String [refinement](https://docs.ruby-lang.org/en/2.4.0/syntax/refinements_rdoc.html). What do you think?

Reply:
Personally, I'm not a big fan of refinements.  In this case, it would need to be on the `Symbol` class, so I'd expect `camelize` to return a symbol, but it needs to be a string.  I'm happy to make the change if you think it'd improve the project, but I'm afraid it makes things less clear and adds too much magic here.

Reply:
Sounds reasonable, it does not sound like an improvement